### ISSUE TITLE
Clearly specify the license of the project

### ISSUE THREAD

**Issue by derberg** [asyncapi/brand#79]
as we keep here all different brand-related graphics we need to make sure it is clear how people can use them from a license perspective.

It is not that obvious for people that IP of the project belongs to Linux Foundation and probably whatever people want they can do with these graphics (as long as they follow guidelines)

- investigate what other LF projects use for license, creative commons or something else and create a proper license file in the repo
- add to readme a clear note, explanation if people can freely use the graphics and if they need to, maybe reference to asyncapi.com or stuff like that

**Comment by Uzume** [comment]
I agree a license should be specified. Being as this is all under the Linux Foundation I am going to assume I can copy these logos, etc. but a license would clearly specify such things.

**Comment by Uzume** [comment]
I agree a license should be specified. Being as this is all under the Linux Foundation I am going to assume I can copy these logos, etc. but a license would clearly specify such things.
